code/modules/ninja/suit/n_suit_verbs/ninja_net.dm

```diff
@@ -1,24 +1,28 @@
 
 //Allows the ninja to kidnap people
-/obj/item/clothing/suit/space/space_ninja/proc/ninjanet(mob/living/carbon/C in oview())//Only living carbon mobs.
+/obj/item/clothing/suit/space/space_ninja/proc/ninjanet()
+	var/mob/living/carbon/human/H = affecting
+	var/mob/living/carbon/C = input("Select who to capture:","Capture who?",null) as null|mob in oview(H)
```

Comment:
If you feel like doing bit more work this would probably be better as ranged ability.